openshift/android-sdk-sample-gluster-pv.json

```diff
@@ -0,0 +1,21 @@
+{
+  "kind": "PersistentVolume",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "android-sdk-pv1"
+  },
+  "spec": {
+    "capacity": {
+      "storage": "10Gi"
+    },
+    "accessModes": [
+      "ReadWriteOnce"
+    ],
+    "persistentVolumeReclaimPolicy": "Recycle",
```

Comment:
According to kubernetes docs "Currently, only NFS and HostPath support recycling."